site/specs/numbers.json

```diff
@@ -1586,7 +1586,7 @@
           "/accounts"
         ],
         "description": "<p>Retrieves information associated with the bulk port-in order specified by the orderId URI parameter.</p>\n",
```

Comment:
We should probably link to the new guide in each of the methods under `/bulkPortins`